admin/src/utils/constants.ts

```diff
@@ -7,6 +7,8 @@ export const APPROVAL_STATUS = {
 export const REPORT_STATUS = {
   RESOLVED: "RESOLVED",
   OPEN: "OPEN",
+  BLOCKED: "BLOCKED",
+  ...APPROVAL_STATUS,
```

Comment:
The spread operator should be used at the beginning of an object. Is this applicable here too?